rust/generate_rust_analyzer.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
I think this would go in `scripts/`.

Also, I think Python would be more readable for templating, but... :-P

Reply:
My bash skills are better than python. So I'm happy if this PR gets superseded by a python implementation. But I'm also quite happy with this implementation.

moving it to `scripts/` sound like a good idea. will do after the `module!` rebuilding

Reply:
A shell script is fine :)

Reply:
Extra note: Python could be nice to automatically format the JSON output (it comes with built-in support for it).